tests/src/FunctionalJavascript/Integration/ParagraphSplitTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Drupal\Tests\thunder\FunctionalJavascript\Integration;
+
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderArticleTestTrait;
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderJavascriptTestBase;
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderParagraphsTestTrait;
+
+/**
+ * Tests the paragraph split module integration.
+ *
+ * @group _Thunder
```

Comment:
So it will stay disabled for now?

Reply:
No, its just work in progress. I just wanted to show early